pdocs/templates/text.mako

```diff
@@ -42,7 +65,18 @@ class ${cls.name}(
 )
 ```
 
+% if hasattr(cls, 'parsed_docstring') and cls.parsed_docsting:
+    # table with arguments info
+    % if cls.parsed_docstring.params:
+| Parameter | type | description | default |
+|---|---|---|---|
+        % for p in cls.parsed_docstring.params:
+| ${p.arg_name} | ${p.type_name} | ${p.description} | ${p.default} |
+        % endfor
+    % endif
+% else:
```

Comment:
We can add the other attributes/sections of parsed_docstring here

Reply:
This `else` block is if `parsed_docstring` doesn't exist. Then we just print the string.

If there are other attributes of `parsed_docstring`, those would go above inside the first `% if` block I think?

Reply:
Yeah, sorry, big thumbs on a somewhat unresponsive smartphone touchscreen :)